src/base.jl

```diff
@@ -1318,7 +1308,7 @@ get_forecast_initial_times(sys::System) = IS.get_forecast_initial_times(sys.data
 """
 Return the total period covered by all forecasts.
 """
-get_forecast_total_period(sys::System) = IS.get_forecast_total_period(sys.data)
+#get_forecast_total_period(sys::System) = IS.get_forecast_total_period(sys.data)
```

Comment:
Let me know if I need to restore this. It is not used in PSI.

Reply:
if at this rate we haven't used it, it isn't needed.